docs/admin-installation.adoc

```diff
@@ -31,18 +31,52 @@ git checkout - && \
 git subtree add --squash --prefix=ods-pipeline subtree-split-branch-$pipelineGitRef
 ```
 
-Once this is done, change to the new folder `ods-pipeline` to configure the values and secrets to use for the installation. It is recommended to encrypt the secrets at rest, therefore the following describes how to use the `helm-secrets` plugin to edit the secrets. For this, you'll need a link:https://docs.github.com/en/github/authenticating-to-github/managing-commit-signature-verification/generating-a-new-gpg-key[GPG keypair], which fingerprint needs to be recorded in a `.sops.yaml`, e.g.:
+Once this is done, change to the new folder `ods-pipeline` to configure the values and secrets to use for the installation.
 
+For the values, just run:
 ```
-creation_rules:
-  - pgp: "A2F9 D4AF 800A 5499 3667 5C66 B92D F67B 0EF2 B057"
+cp chart/values.yaml values.yaml
 ```
 
-Now copy the default values files:
+Regarding the secrets, it is recommended to encrypt them at rest, therefore the following describes how to first encrypt them, but also how to edit them using the `helm-secrets` plugin.
+
+For this, you'll need to install `link:https://github.com/mozilla/sops[sops]` and `link:https://github.com/FiloSottile/age[age]` for en-/decryption of the `secrets.yaml` file.
+
+As described in the `sops` link:https://github.com/mozilla/sops#22encrypting-using-age[documentation], when decrypting
```

Comment:
not sure how it happened that I did not put the right link, but the mozilla link is actually without `22` in it: https://github.com/mozilla/sops#encrypting-using-age
I need to also fix this in the user-installation doc.

Reply:
feel free to update it here with a new commit 👍

Reply:
while doublechecking I found out that it works if you press the link but not if reloading being on the page already. I guess that might be standard behaviour of these anchor links? Anyway, it can stay as it is 👍🏻